app/controllers/api/storage.php

```diff
@@ -21,8 +22,121 @@
 use Appwrite\OpenSSL\OpenSSL;
 use Appwrite\Utopia\Response;
 use Utopia\Config\Config;
+use Utopia\Validator\Integer;
 use Utopia\Database\Query;
-use Utopia\Validator\Numeric;
+
+App::post('/v1/storage/buckets')
```

Comment:
We should add a webhooks test for this endpoint as well, it helps us check the relevant event is actually executed and that the payload is correct.

Reply:
Webhooks test added and passing locally